### ISSUE TITLE
UI for content writer

### ISSUE THREAD

**Issue by galexandre** [LiffeyValleyAC/LiffeyValleyAC.github.io#293]
Hi all,
I, recently, searched for a couple of existing sotfwares to simplify the creation of content. I found a post with a list and I will try some of them and do a POC with one of my project to select the best option (the easiest to implement, the most user-friendly and the less expensive).
I will keep this issue updated with the list of software tested.

**Comment by mgriffin** [comment]
Netlify CMS might be a good one to look into, I started to look into it in https://github.com/LiffeyValleyAC/LiffeyValleyAC.github.io/issues/164 but never went anywhere with it at the time

**Comment by galexandre** [comment]
Yes, I saw that one in a list somewhere. I saw a plugin that can be associated to the current installation of Jekyl. I might try that one this weekend.

**Comment by galexandre** [comment]
- https://jekyll.github.io/jekyll-admin/ - cannot be installed as it is not part of the plugin whitelist https://pages.github.com/versions/

**Comment by galexandre** [comment]
Other options I found
- https://prismic.io/
- https://www.contentful.com/
- https://www.contentful.com/
- https://www.siteleaf.com/plans/
